### ISSUE TITLE
[Widget UI] Hide the character counter when the composer is hidden.

### ISSUE THREAD

**Issue by ear-dev** [WideChat/Rocket.Chat#1164]
Currently we still see the character counter, even though the composer is no longer available after a chat session has been closed.

**Comment by AlexanderKanakis** [comment]
PRs: 
https://github.com/WideChat/Rocket.Chat.Livechat/pull/142

**Comment by ear-dev** [comment]
@AlexanderKanakis I am moving this back into the In Progress column while you work on the enhancement to tie the counter to the typing_enabled state.

**Comment by AlexanderKanakis** [comment]
@ear-dev do we want it on the same PR?

**Comment by ear-dev** [comment]
> @ear-dev do we want it on the same PR?

Yes, I think it's just one feature, so it should be in the same PR please.  Thanks.
